forte_wrapper/spacy/spacy_processors.py

```diff
@@ -46,13 +47,27 @@ def set_up(self):
             download(self.lang_model)
             self.nlp = spacy.load(self.lang_model)
 
+        # pylint: disable=import-outside-toplevel
+        if 'ent_link' in self.processors:
+            from scispacy.linking import EntityLinker
+            linker = EntityLinker(resolve_abbreviations=True, name="umls")
```

Comment:
Will this automatically download the model like spacy?

Reply:
It depends. For some models it can automatically download, for scispacy models we used for medical entity linking, the only way to install it is using pip. Here is a more detailed link https://github.com/explosion/spaCy/issues/6458

Reply:
Then we need to install the model somewhere. Two suggestions:

1. Call the pip install model download in this setup section.
2. Do it in the setup.py, alongside the scispacy requirements.

Either is fine, the goal is to do our best to reduce any manual work from the user.